Source/Charts/Data/Implementations/Standard/PieChartDataSet.swift

```diff
@@ -84,6 +84,9 @@ open class PieChartDataSet: ChartDataSet, IPieChartDataSet
     /// When valuePosition is OutsideSlice, indicates line color
     open var valueLineColor: NSUIColor? = NSUIColor.black
 
+    /// When valuePosition is OutsideSlice and enabled, line will have the same color as the slice
+    open var matchValueLineColorToPieSlice: Bool = false
```

Comment:
I was hoping we can use a shorter name

Reply:
Wanted it to be descriptive.
What about `valueLineAutoColor`?

Reply:
haha, I'm no native speaker. @jjatie @petester42 ?

Reply:
I am willing to change the name to whatever you suggest. 
I agree it was very long!